templates/content/stats.html.twig

```diff
@@ -0,0 +1,69 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+
+    <div class="jumbotron">
+        <h1>Ho, ho, ho, Here is the statistics of Secret Santa</h1>
+    </div>
+
+    <div class="container-charts">
+        <h3 class="graph-title">Number of Secret Santa per month</h3>
+        <div class="chart chart-month">
+            {% if statisticsCounter.hashesMonth %}
+                {% set values = statisticsCounter.hashesMonth %}
+                {% for value, key in values %}
+                    <div  class="graph-column" style="width: {{ (1 / values|length) * 750 }}px;padding: 0 10px;">
+                        <div class="graph-counterValue">{{ key }}</div>
+                        <div class="graph-bar" style="width: {{ (1 / values|length) * 750 }}px; height: {{ (key / max(values)) * 500 }}px;"></div>
+                        <div class="graph-legend-x">{{ value|replace({'date:' : ''}) }}</div>
+                    </div>
+                {% endfor %}
+            {% else  %}
+                <div class="graph-noData">Sorry, there is no statistic for this section 🙈</div>
+            {% endif %}
+        </div>
+
+        <h3 class="graph-title">Number of Secret Santa per year</h3>
+        <div class="chart chart-year">
+            {% if statisticsCounter.hashesYear %}
+                {% set values = statisticsCounter.hashesYear %}
+                {% for value, key in values %}
+                    {% if value matches '/date:\\d+$/'  %}
```

Comment:
why do we need this condition?

Reply:
This serves to check if there is any stats.
If not it shows some message

Reply:
You can remove the condition.